release-team/role-handbooks/enhancements/README.md

```diff
@@ -64,50 +61,73 @@ The shadows should be selected keeping in mind that one of them may eventually b
+- Ping any owners on `lifecycle/frozen` label from all issues to see if there is any planned work or if the issue is no longer relevant to Kubernetes.
+- Send an email to the [Kubernetes-Dev](https://groups.google.com/forum/#!forum/kubernetes-dev) mailing list with a call for enhancements. View prior examples for references ([1](https://groups.google.com/forum/#!msg/kubernetes-dev/vGSBzUUWp9g/bFa1re3ZAgAJ) [2](https://groups.google.com/forum/#!msg/kubernetes-dev/T-kIHtgS5J4/ZjKTsCf_AwAJ;context-place=msg/kubernetes-dev/vGSBzUUWp9g/bFa1re3ZAgAJ)).
+
+
+#### Pre-Enhancements Freeze (Week 1)
+
+- Go to every open issue that is marked with `tracked/no` and ping the owner to see if the enhancement should be included in the current milestone. If there is no graduation or major change, leave as-is. If the enhancement is graduating or undergoing a major change, remove the `tracked/no` label and add `tracked/yes`, add the issue to the current milestone, and set it's status to `Tracked` within the tracking sheet if it meets the requirements. If it is missing some  of the requirements (e.g. KEP in a provisional state), let the issue owner know and add it as `At Risk`. Another great task for shadow help or some automation.
+- Verify issues have k/k PRs associated so they can be referenced and easily tracked. This is going to be critical come Enhancement Freeze and Code Freeze to see the status of the code.
+- Work with the Release Lead to attend every SIG meeting over the span of 2-3 weeks. Take 2 minutes to introduce yourself, talk about release information, and relay information about issues that haven't been removed from the previous milestone or call out issues that may need some attention.
+- New issues may be added during the release cycle. Take the opportunity to apply the correct labels, milestone information, and add them to the `data` tab within the sheet. If it's not being tracked for the current release, adding the information will help a later release team.
+- Weekly Release meetings require updates of current status. Use the `Feature stats` tab to update the release team on counts of enhancements in good and bad progress.
+- Start reminding Issue owners that KEPs are required for each enhancement and that KEPs must be in an implementable state by Enhancement Freeze.
+
+
+#### Collection Monitoring and Triage (Week 2-3)
+
+- Stay on top of comments in issues when owners respond. Apply correct labels, milestone information, update their status in the sheet if necessary.
+- Mark features as `At Risk` if there is no communication, active PRs on the issues, or it is missing other requirements coming into Enhancement Freeze.
+- Start syncing with Communications Team on giving an induction what's coming up for the release.
```

Comment:
induction -> indication/signal

Reply:
It was actually meant to be an induction. The enhancements lead/team is supposed to sync up with the comms lead to give a brief about the important enhancements in the milestone.